presto-product-tests/README.md

```diff
@@ -107,16 +107,32 @@ following command:
 
 ```
 ./mvnw install -DskipTests
-presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query
+presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector
 ```
 
-where `<profile>` is one of either:
+where [profile](#profile) is one of either:
 - **multinode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a distributed Presto installation running on
- multiple Docker containers.
+ multiple Docker containers. For multinode the default configuration is 1 coordinator and 1 worker.
 - **singlenode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a single node installation of Presto also running
- on a single Docker container.
+ on a single Docker container. With no impersonation presto accesses hdfs as the user mentioned in -DHADOOP_USER_NAME.
```

Comment:
@ebd2 - That links to teradata docs 141t. I think, it should be linked to facebook docs.